plugin/completion/flags_manager.py

```diff
@@ -204,11 +204,11 @@ def get_flags(self):
 
         if self._clang_complete_file.was_modified():
             log.debug(" .clang_complete modified. Load new flags.")
-            self._flags = FlagsManager.flags_from_clang_file(
-                self._clang_complete_file)
+            generated_flags = list(FlagsManager.flags_from_clang_file(
+                self._clang_complete_file))
```

Comment:
merge_flags will probably throw if this branch is not entered. Should initialize `generated_flags` outside of 'if'.